lib/components/user/monitored-trip/trip-summary-pane.tsx

```diff
@@ -1,61 +1,234 @@
-import { FormattedList, FormattedMessage } from 'react-intl'
+import { Bell, BellSlash, Calendar, Clock } from '@styled-icons/fa-regular'
+import { FormattedMessage, useIntl } from 'react-intl'
 import React from 'react'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
 import { MonitoredTripProps } from '../types'
-import FormattedDayOfWeek from '../../util/formatted-day-of-week'
-import Strong from '../../util/strong-text'
+import LocationIcon from '@opentripplanner/location-icon'
+import styled from 'styled-components'
 
-import TripSummary from './trip-summary'
+import { InlineLoading } from '../../narrative/loading'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+
+import MonitoredDays from './trip-monitored-days'
+import TripSummary from './trip-duration-summary'
 
 /**
  * Displays the summary information of a monitored trip.
  */
+
+const SavedTripBody = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 0 0 0 15px;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+    padding: 0;
+  }
+`
+
+const LocationDetails = styled.div`
+  align-items: start;
+  border-right: 1px solid #ddd;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  padding: 20px;
+  padding-right: 30px;
+  width: 50%;
+
+  @media (max-width: 768px) {
+    align-items: center;
+    border-bottom: 1px solid #ddd;
+    border-right: 0;
+    border-top: 1px solid #ddd;
+    padding: 30px 20px 30px 20px;
+    width: 100%;
+  }
+`
+const ItineraryDetails = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 20px;
+  padding-left: 30px;
+
+  @media (max-width: 768px) {
+    padding: 30px 20px 20px 20px;
```

Comment:
Total nitpick. You already set padding so you can embrace the cascade 
```suggestion
    padding-top: 30px;
    padding-left: 30px;
```